docs/storage.md

```diff
@@ -422,8 +421,29 @@ Usually, you can use a NFS or SMB share. But if you don't have any of those, you
 That's it!
 
 :::tip
-Don't forget to rescan your SR after adding ISO files. Rescan is done automatically every 10 minutes otherwise.
-:::
+Don't forget to rescan your SR after adding, changing, or deleting ISO files. Rescan is done automatically every 10 minutes otherwise.:::
```

Comment:
Idem

Reply:
Not sure I follow IDEM.  Does that mean "me too"?  In any case I don't know what to do with this comment other than 😄